src/core/mount.c

```diff
@@ -1484,9 +1485,15 @@ static void mount_sigchld_event(Unit *u, pid_t pid, int code, int status) {
                 mount_enter_dead(m, f);
                 break;
 
-        case MOUNT_MOUNTING_DONE:
+        case MOUNT_MOUNTING_DONE: {
+                const MountParameters *p = get_mount_parameters(m);
```

Comment:
I am not sure whether `get_mount_parameters()` or `get_mount_parameters_fragment()` is correct here.